code/game/turfs/simulated/walls_misc.dm

```diff
@@ -92,24 +94,6 @@
 		animate(src, color = previouscolor, time = 8)
 		addtimer(CALLBACK(src, /atom/proc/update_atom_colour), 8)
 
-/turf/simulated/wall/clockwork/dismantle_wall(devastated=0, explode=0)

```

Comment:
because when taking a wall apart it created a hole into space

Reply:
The trouble here that sounds like what happened is that `baseturf` might be set to `/turf/space`. What you want to do to get around that is put down a `baseturf_helper` of the appropriate type (beach/sand or beach/water), which should set the `baseturf` of all tiles in the area it's placed in to what's appropriate.

Reply:
no no I made sure to test deconstructing the brass wall on station. When I did it lead to space. Removing the dismantle_wall line fixed that and it lead to floor plating like it should.

Reply:
oh yeah. looking at this further, this is basically a copypasta of regular wall code. You have my full endorsement to tear this out, particularly since I figure these walls would normally only be built on top of plating.